Failed: HubSpec

https://jenkins.akka.io:8498/job/akka-multi-node-nightly/5670/testReport/junit/akka.stream.scaladsl/HubSpec/PartitionHub_must_properly_signal_error_to_consumers/


```
Error Message
java.lang.AssertionError: assertion failed: timeout (3 seconds) during expectMsg while waiting for OnNext(15)
Stacktrace
sbt.ForkMain$ForkError: java.lang.AssertionError: assertion failed: timeout (3 seconds) during expectMsg while waiting for OnNext(15)
	at scala.Predef$.assert(Predef.scala:170)
	at akka.testkit.TestKitBase$class.expectMsg_internal(TestKit.scala:401)
	at akka.testkit.TestKitBase$class.expectMsg(TestKit.scala:378)
	at akka.testkit.TestKit.expectMsg(TestKit.scala:890)
	at akka.stream.testkit.TestSubscriber$ManualProbe$$anonfun$expectNextN$1.apply(StreamTestKit.scala:387)
	at akka.stream.testkit.TestSubscriber$ManualProbe$$anonfun$expectNextN$1.apply(StreamTestKit.scala:387)
	at scala.collection.Iterator$class.foreach(Iterator.scala:891)
	at scala.collection.AbstractIterator.foreach(Iterator.scala:1334)
	at scala.collection.IterableLike$class.foreach(IterableLike.scala:72)
	at scala.collection.AbstractIterable.foreach(Iterable.scala:54)
	at akka.stream.testkit.TestSubscriber$ManualProbe.expectNextN(StreamTestKit.scal
```

--- a/akka-stream-tests/src/test/scala/akka/stream/scaladsl/HubSpec.scala
+++ b/akka-stream-tests/src/test/scala/akka/stream/scaladsl/HubSpec.scala
@@ -315,7 +315,7 @@ class HubSpec extends StreamSpec {
 
       // sending the first element is in a race with downstream subscribing
       // give a bit of time for the downstream to complete subscriptions
-      Thread.sleep(50)
+      Thread.sleep(100)
 
       (1 to 8) foreach (upstream.sendNext(_))
 
@@ -582,7 +582,7 @@ class HubSpec extends StreamSpec {
 
       // to make sure downstream subscriptions are done before
       // starting to send elements
-      Thread.sleep(50)
+      Thread.sleep(100)
 
       (0 until 16) foreach (upstream.sendNext(_))
 

